lib/LambdaHelper.js

```diff
@@ -68,7 +65,13 @@ var _lambdaize = function (userFunc) {
             func.apply(this, event.args);
         }
         catch(error) {
-            _newCallback({isFaulty: true, stack: error.stack, message: error.message});
+            var objectifyError = function(err) {
```

Comment:
Should this function be outside the catch block so it doesn't get instantiated every time there is an error ?